src/v3/src/transformer/button/transform.test.ts

```diff
@@ -39,9 +39,6 @@ jest.mock('./transformRegisterButton', () => ({
 jest.mock('./transformCancelButton', () => ({
   transformCancelButton: () => () => ({}),
 }));
-jest.mock('./transformLaunchAuthenticatorButton', () => ({
```

Comment:
removed from this file since transformLaunchAuthenticatorButton file is no longer used